cg/constants/gene_panel.py

```diff
@@ -41,6 +41,7 @@ class GenePanelMasterList(StrEnum):
     SOVM: str = "SOVM"
     STROKE: str = "STROKE"
     AID: str = "AID"
+    INHERITED_CANCER: str = "Inherited cancer"
```

Comment:
This one seems to deviate from the naming pattern of the ones above, is that intended?

Reply:
I also thought it was strange and that's why I asked @northwestwitch about it, but this ID seems to have been chosen by the customers as such so it's intended.
This is the panel created by the customer, for reference: https://scout.scilifelab.se/panels/61ee70b121c519723d3f3049